server/plugin/api.go

```diff
@@ -1062,12 +1092,18 @@ func (p *Plugin) getIssueByNumber(c *UserContext, w http.ResponseWriter, r *http
 
 	githubClient := p.githubConnectUser(c.Context.Ctx, c.GHInfo)
 
-	result, _, err := githubClient.Issues.Get(c.Ctx, owner, repo, numberInt)
-	if err != nil {
+	var result *github.Issue
+	if cErr := p.useGitHubClient(c.GHInfo, func(info *GitHubUserInfo, token *oauth2.Token) error {
+		result, _, err = githubClient.Issues.Get(c.Ctx, owner, repo, numberInt)
+		if err != nil {
+			return err
+		}
+		return nil
+	}); cErr != nil {
 		// If the issue is not found, it's probably behind a private repo.
 		// Return an empty repose in this case.
 		var gerr *github.ErrorResponse
```

Comment:
What is the use of gerr?

Reply:
Converting API error to github.ErrorResponse